src/asciidoc/docs/platform-features/richPreview.adoc

```diff
@@ -1,143 +1,61 @@
-By default, the platform takes a screenshot of the initial view.
+When you share a Koji app on social media, the Koji platform renders a thumbnail (Open Graph) image based on a screenshot of the app.
+If you were designing a web app from scratch, you would use meta tags, such as `og:image`, `og:title`, and `og:description`, to set properties for the thumbnail.
+The Koji platform automatically sets these properties for you, but you can override some of Koji's default settings to provide a customized thumbnail.
 
-In some cases, the initial view might not be the best preview image for your template.
-For example, if a game template starts with an instruction screen, a screenshot of the first level might provide a better preview image.
-To improve the sharing experience in these cases, you can render a custom view of your template that is designed specifically to make an impression when sharing a remix.
+By default, the platform takes a screenshot of the initial view.
+In some cases, the initial view might not be the best preview image for your app.
+For example, if a game app starts with an instruction screen, a screenshot of the first level might provide a better preview image.
+To improve the sharing experience in these cases, you can render a custom view of your app that is designed specifically to make an impression when sharing it.
 You can also control the timing of the screenshot to ensure that all relevant assets have loaded before the preview image is generated.
 
-You can also customize the sharing metadata by configuring an entitlement.
+You can also give creators the ability to customize the thumbnail by enabling <<entitlements#_custom_metadata,custom metadata>>.
 
-== Customizing the sharing image
+== Customizing the thumbnail
 
-To customize the sharing image for your template, render a custom view when the URL contains the `koji-screenshot=1` query parameter.
-For optimal rich preview support, design this view with the following considerations in mind.
+To customize the thumbnail for your app, render a custom view when the URL contains the `Koji.playerState.context=screenshot` query parameter.
+For optimal thumbnail support, design this view with the following considerations in mind.
 
 * The image is responsive.
 * The image looks good with a square aspect ratio.
 * The image is optimized for faster loading.
 For example, to reduce the image size to 400x400 pixels and load only the first frame of a gif, you can add query parameters at the end of your image's URL: `\https://images.koji-cdn.com/path/to/myimage.gif?fit=bounds&width=400&height=400&frame=1`
-* The image includes customizable elements to generate unique previews of each remix.
+* The image includes customizable elements to generate unique previews for each customized version of the app.
 
-image::Koji-screenshot=1.svg[Render a view for the rich preview image]
+image::Koji-screenshot=1.png[Render a view for the thumbnail image]
```

Comment:
Remove the `?` before the `Koji.playerState.context` property. It's no longer a query parameter (your code sample uses the property correctly). Also, the text appears blurry.

Reply:
The text doesn't look blurry to me. Can you check again? Maybe your screen hadn't finished rendering or something.